<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>540</width>
       <height>320</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>540</width>
       <height>320</height>
      </size>
     </property>
     <property name="title">
      <string>身份证信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="8" column="1" rowspan="2" colspan="3">
       <widget class="QLabel" name="label_Address_2">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>42</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLabel" name="label_Birthday">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>1980年10月11日</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 85, 255);</string>
        </property>
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_11">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 85, 255);</string>
        </property>
        <property name="text">
         <string>指纹信息</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 85, 255);</string>
        </property>
        <property name="text">
         <string>出生日期</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_Gender">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>女</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" rowspan="2" colspan="3">
       <widget class="QLabel" name="label_Address">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>新疆维吾尔族自治区乌鲁木齐市阿拉木镇乌干比达</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 85, 255);</string>
        </property>
        <property name="text">
         <string>签发机关</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLabel" name="label_Name">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>东方不败</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 85, 255);</string>
        </property>
        <property name="text">
         <string>住址</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_10">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 85, 255);</string>
        </property>
        <property name="text">
         <string>有效期限</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 85, 255);</string>
        </property>
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QLabel" name="label_Identify">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>431233198010114521</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 85, 255);</string>
        </property>
        <property name="text">
         <string>身份证号码</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 85, 255);</string>
        </property>
        <property name="text">
         <string>民族</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3">
       <widget class="QLabel" name="label_IssueAuthority">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>新疆维吾尔族自治区乌鲁木齐市阿拉木镇公安局</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="3">
       <widget class="QLabel" name="label_ExpirationDate">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>2010.10.31-2030.10.31</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_Nation">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>新疆维吾尔族</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="7" colspan="2">
       <widget class="QLabel" name="label_Photo">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>160</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>102</width>
          <height>126</height>
         </size>
        </property>
        <property name="text">
         <string>照片</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>248</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Port">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>自动检测</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>串口</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Start">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>开始读取</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Start_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>OpenPort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Stop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>停止读取</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Information">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
